02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,27 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
+
+  <title>My First Webpage</title>
   <link rel="stylesheet" href="css/style.css">
+
 </head>
+
 <body>
 
+   <h1>Goodbye, space!</h1>
```

Comment:
I like the indentation and use of many elements. You seem to have a grasp of this, so I'd like to see more. Try to use more elements like <table>, <tr>, <td>, <ul>, <li>. Great beginning!